R/ff_predict.R

```diff
@@ -112,7 +112,7 @@ load_model <- function(model) {
 
     if (file.exists(rda_file)) {
       model_features <- get(load(rda_file))
-      attr(model, "feature_names") <- model_features
+      model$feature_names <- model_features
     }
```

Comment:
was there any reason you changed the syntax here? since the previous one is more suggested?

Reply:
the first one did not work, the second one did. if i returned model from outside this function it would not show feature_names as an attribute with the first method, with the second it would